bybit.py

```diff
@@ -36,23 +36,24 @@
 def get_order_book(pair=PAIR, level=LEVEL):
     """
     Returns L2 order book
-    TODO: might want to add depth level as parameter
     """
     api_command = API_LINK + f'orderBook/{level}?symbol={pair}'
     logger.info(f'GET #{api_command}')
     resp = requests.get(api_command).json()
     if resp['ret_code'] == 0: return resp['result']
     return resp['ret_msg']
 
-# TODO: everything else
-
-def get_trades(pair=PAIR, since):
-    """Returns last 1000 trades by default"""
-    api_command = API_LINK + f'Trades?pair={pair}&since={since}'
+def get_trades(pair=PAIR):
+    """
```

Comment:
Could easily overload storage. Will probably need to get last N trades. 
If possible the trades in the last N seconds or so could also work.